packages/ui-bindings/Unit/getUnitById.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * A method to get all Units
```

Comment:
I think this may have been a copy+paste job you forgot to change because this is for getting a single unit, right?